### PR TITLE
Import moment locales to Indonesia and Vietnamese configs

### LINKED ISSUE #4004: Update config files with correct locales for 3 new services
**Is your feature request related to a problem? Please describe.**
Timestamps are not appearing in the correct locale for the following services:
- Afrique
- Indonesia
- Vietnamese

The timestamp is currently appearing in en-GB for these services, as the locales referenced do not exist.

**Describe the solution you'd like**
Timestamps appear in the correct locale for the service. Service and locale below:

- Afrique = fr
- Indonesia = id
- Vietnamese = vi

Reference: https://docs.google.com/spreadsheets/d/1mfeOBq_BSqDAWipDtOozs3x4eh1sdrkrXdToVpKJ1oY/edit#gid=455871924 and https://bbc.github.io/psammead/?path=/info/utilities-psammead-locales--moment-afaan-oromoo-om

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by lizcameron**
Resolves https://github.com/bbc/simorgh/issues/4004

**Overall change:** The configs for Indonesia and Vietnamese are missing an import statement for the moment locales. This change adds them.

**Code changes:**

- Add import 'vi' locale into Vietnamese config
- Add import 'id' locale into Indonesia config

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by samora — COMMENTED** [review]
Looks good to me. Is there any way to test this?

**Comment by jimjohnsonrollings** [comment]
Before this change is applied, the client-render of the article in these services had the timestamp in English (but the server-render did not).  After this change both should show the timestamp in the correct language.

**Review by 12 — APPROVED** [review]
Looks good code-wise and in browser (according to https://github.com/bbc/simorgh/pull/4059#issuecomment-537505625) 👍

**Review by dr3 — APPROVED** [review]

File: src/app/lib/config/services/afrique.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
 import { latinDiacritics } from '@bbc/gel-foundations/scripts';
 import { afrique as brandSVG } from '@bbc/psammead-assets/svgs';
 import '@bbc/moment-timezone-include/tz/GMT';
+import 'moment/locale/fr';
```
Comment (dr3) [inline src/app/lib/config/services/afrique.js]:
```suggestion
```

**Comment by dr3** [comment]
Afrique was just done in https://github.com/bbc/simorgh/pull/4074

**Review by FK78 — APPROVED** [review]

**Pull request merged**